src/main/kotlin/com/ecwid/apiclient/v3/ApiClient.kt

```diff
@@ -250,6 +250,7 @@ interface BatchApiClient {
 interface ProductVariationsApiClient {
 	fun createProductVariation(request: CreateProductVariationRequest): CreateProductVariationResult
 	fun uploadVariationImage(request: ProductVariationImageUploadRequest): ProductVariationImageUploadResult
+	fun uploadProductVariationImageAsync(request: ProductVariationImageAsyncUploadRequest): ProductVariationImageAsyncUploadResult
```

Comment:
We already have method uploadVariationImage, let we create async with name uploadVariationImageAsync?

ProductVariationImageAsyncUploadRequest and ProductVariationImageAsyncUploadResult  we leave as it is like above